landscape.yml

```diff
@@ -7012,6 +7012,13 @@ landscape:
               youtube_url: https://www.youtube.com/channel/UC4pLFely0-Odea4B2NL1nWA
               slack_url: https://freenode.net/
```

Comment:
This is incorrect

Reply:
Yes, needs to be updated to `https://libera.chat/`. @roidelapluie are you already working on a PR?

Reply:
Apparently not. I'll send one in a minute.

Reply:
https://github.com/cncf/landscape/pull/2899